### ISSUE TITLE
Main Autotiling cycle sizes

### ISSUE THREAD

**Issue by j-mir-prazak** [gTile/gTile#405]
Hi, I used to use gTile main autotile shortcut to cycle through different sizes by pressing it multiple times. For several months now I was not able to. However it seems like it might still be working and you still have it on demo-videos. It it working? Is something wrong with my system?

Thanks!

**Comment by schnz** [comment]
Hey there,

I suppose the confusion might raise from the fact that you likely refer to the "resize presets" which are different from "autotiling". But  the feature is still there. Please check the extension settings under the corresponding tab. If it still doesn't work for you - please report back.

**Comment by j-mir-prazak** [comment]
sorry, i dont know how better to refer to it:
https://github.com/gTile/gTile/assets/3457747/40489511-1c5d-44da-b59f-9445cf50b86c

i mean this - not resize preset; but autotiling - in the video, after pressing M multiple times, the size of the main window changes. I know it used to work for me like that. after some gnome update and related gtile update, it all of the sudden did not work

thank you for your response

**Comment by schnz** [comment]
Oh - not your fault then!

gTile has undergone a rewrite in order to be compatible with Gnome 45. I suppose this is when you started noticing that the shortcut wasn't working anymore like before. 

Yeah but long story short: I did rewrite the entire extension and I tried really hard to implement (most) things like they were before. Obviously I missed out on that one because - until now - I wasn't even aware that this was a thing. 

So this is definitely an _unwanted_ regression then. Shouldn't be too much work to implement though. I'll come back to you.
